tests/phpunit/CRM/Dedupe/MergerTest.php

```diff
@@ -1472,6 +1474,26 @@ public function testMergeWithDeclaredSearchJoin(): void {
     $this->callAPISuccess('Contact', 'merge', ['to_keep_id' => $contact1, 'to_remove_id' => $contact2]);
   }
 
+  /**
+   * Test that the presence of a custom field attached to the relationship does not block merge.
+   */
+  public function testMergeWithRelationshipWithCustomFields(): void {
+    $contact1 = $this->individualCreate();
+    $contact2 = $this->individualCreate();
+    $this->createCustomGroupWithFieldOfType(['extends' => 'Relationship']);
+    $this->callAPISuccess('Relationship', 'create', [
+      'contact_id_a' => $contact2,
+      'contact_id_b' => CRM_Core_Config::domainID(),
```

Comment:
@eileenmcnaughton is this a proper contact id value?

Reply:
hmm accidentally yes.....